test/unit/node/generated-code-dependencies.js

```diff
@@ -347,17 +347,26 @@ suite('context test', function() {
     });
   });
 
+  function checkFile(outDir, goldenDir, filename) {
+    var actualContents = fs.readFileSync(path.resolve(outDir, filename));
+    var actual = actualContents.toString().split('\n');
```

Comment:
I added a single character to on of the golden files. With the current splitting:
```
  1) context test compile module dir option AMD:

      Uncaught AssertionError: expected '  X"use strict";' to equal '  "use strict";'
      + expected - actual

      +  "use strict";
      -  X"use strict";
```
With `assert.deepEqual(actualContents.toString(), expectedContents.toString());`:
```
1) context test compile module dir option AMD:

      Uncaught AssertionError: expected 'define(["./dep"], function($__0) {\n  "use strict";\n  if (!$__0 || !$__0.__esModule)\n    $__0 = {default: $__0};\n  var q = $__0.q;\n  var p = \'module\';\n  return {\n    get p() {\n      return p;\n    },\n    __esModule: true\n  };\n});\n' to deeply equal 'define(["./dep"], function($__0) {\n  X"use strict";\n  if (!$__0 || !$__0.__esModule)\n    $__0 = {default: $__0};\n  var q = $__0.q;\n  var p = \'module\';\n  return {\n    get p() {\n      return p;\n    },\n    __esModule: true\n  };\n});\n'
      + expected - actual

      +"define([\"./dep\"], function($__0) {\n  X\"use strict\";\n  if (!$__0 || !$__0.__esModule)\n    $__0 = {default: $__0};\n  var q = $__0.q;\n  var p = 'module';\n  return {\n    get p() {\n      return p;\n    },\n    __esModule: true\n  };\n});\n"
      -"define([\"./dep\"], function($__0) {\n  \"use strict\";\n  if (!$__0 || !$__0.__esModule)\n    $__0 = {default: $__0};\n  var q = $__0.q;\n  var p = 'module';\n  return {\n    get p() {\n      return p;\n    },\n    __esModule: true\n  };\n});\n"
```
The split seems better in this comparison. Did you have something else in mind?